registries-operator-image/registries-operator-image.kiwi.ini

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: _PRODUCT_/dex-operator:%%LONG_VERSION%% _PRODUCT_/registries-operator:%%LONG_VERSION%%-<RELEASE> -->
```

Comment:
Be careful! this will collide with the dex-operator image. I guess it needs to be registries-operator instead dex-operator

Reply:
thank you, fixed